bundles/org.dataflowanalysis.analysis/META-INF/MANIFEST.MF

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
-Bundle-Name: Data Flow Confidentiality Analysis
-Bundle-SymbolicName: org.palladiosimulator.dataflow.confidentiality.analysis
+Bundle-Name: Data Flow Analysis
+Bundle-SymbolicName: org.dataflowanalysis.analysis
 Bundle-Version: 1.0.0.qualifier
 Automatic-Module-Name: org.palladiosimulator.dataflow.confidentiality.analysis
```

Comment:
automatic-module-name also needs to be changed

Reply:
Renamed all three automatic-module-names 👍🏼